frontend/src/scenes/dashboard/dashboardLogic.js

```diff
@@ -282,56 +291,13 @@ export const dashboardLogic = kea({
                 }),
             })
         },
-
         updateItemColor: ({ id, color }) => {
             api.update(`api/insight/${id}`, { color })
         },
-
-        enableWobblyDragging: () => {
-            clearDOMTextSelection()
-            window.setTimeout(clearDOMTextSelection, 200)
-            window.setTimeout(clearDOMTextSelection, 1000)
-
-            if (!cache.draggingToastId) {
-                cache.draggingToastId = toast(
-                    <>
-                        <p className="headline">Rearranging panels!</p>
-                        <p>
-                            <Link onClick={() => actions.disableDragging()}>Click here</Link> to stop.
-                        </p>
-                    </>,
-                    {
-                        autoClose: false,
-                        onClick: () => actions.disableDragging(),
-                        closeButton: false,
-                        className: 'drag-items-toast',
-                    }
-                )
-            }
-        },
-        enableDragging: () => {
-            if (cache.draggingToastId) {
-                toast.dismiss(cache.draggingToastId)
-                cache.draggingToastId = null
-            }
-        },
-        disableDragging: () => {
-            if (cache.draggingToastId) {
-                toast.dismiss(cache.draggingToastId)
-                cache.draggingToastId = null
-            }
-        },
-        refreshDashboardItem: async ({ id }, breakpoint) => {
-            const dashboardItem = await api.get(`api/insight/${id}`)
-            await breakpoint()
-            dashboardsModel.actions.updateDashboardItem(dashboardItem)
-            if (dashboardItem.refreshing) {
-                setTimeout(() => actions.refreshDashboardItem(id), 1000)
-            }
-        },
         refreshAllDashboardItems: async (_, breakpoint) => {
             await breakpoint(200)
             dashboardItemsModel.actions.refreshAllDashboardItems({})
+            eventUsageLogic.actions.reportDashboardRefreshed(values.lastRefreshed.toString())
```

Comment:
This throws an error when (I think) the dashboard didn't have a created at value so the last refreshed is the same?

See deployed env.
![Uploading Screenshot 2021-03-10 at 09.40.39.png…]()